### ISSUE TITLE
Doesn't load home page

### ISSUE THREAD

**Issue by spottedmahn** [Gimly/NetCoreAngularAzureB2CMsal#1]
Hey @Gimly - when I visit the home page I get

![image](https://user-images.githubusercontent.com/14079228/31792077-cc83ece6-b4e8-11e7-8823-9d37bbb42b5f.png)

Maybe there are some changes do didn't push yet?  I assume it is working for you...

BTW, I think it is great you're adding B2C on top of the Angular template!!

**Comment by Gimly** [comment]
Oops, my bad, I had added (manually) msal.js to the wwwroot/dist folder, forgot that that folder was ignored by git. It should be working correctly now.

I've moved it to a lib folder and updated the msal.js. A better option would be to add the move to the build process, but I don't have time now, will do later.

Be careful as this was meant as a sample project for testing purpose, I didn't manage to make it work quite yet. There is an issue with the initialisation of msal that makes everything fail for some reason.
